tests/integration_test/firebase_auth/firebase_auth_user_e2e_test.dart

```diff
@@ -39,6 +39,27 @@ void main() {
           expect(token?.length, greaterThan(24));
         });
 
+        test('should return a token after signOut() & signIn()', () async {
+          // Demonstrate fix for this issue works: https://github.com/firebase/flutterfire/issues/11297
+          String email = generateRandomEmail();
+
+          await FirebaseAuth.instance.createUserWithEmailAndPassword(
+            email: email,
+            password: testPassword,
+          );
+
+          await FirebaseAuth.instance.signOut();
```

Comment:
Is there any reason why you are first signing in and then out to reproduce this? Are you not getting this issue if you don't do signIn&signOut first?
Because I can reproduce the issue using any credential, no signout required.

Reply:
Correct - I've updated the test.